authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -242,11 +242,20 @@ public SecurityFilterChain unprotectedApiAuthSecurityFilterChain(
   public SecurityFilterChain protectedUnhandledPathsSecurityFilterChain(
       final HttpSecurity httpSecurity) throws Exception {
     // all resources not yet explicitly handled by any previous chain require an authenticated user
-    // thus by default unhandled paths are always protected
+    // thus by default unhandled paths will always return 404 not found
```

Comment:
💭   I would split the fact that we deny access to all unhandles paths + the conversion to 404, see next comment
```suggestion
    // thus by default access to unhandled paths will always be denied
```